packages/webview_flutter/webview_flutter/lib/src/navigation_delegate.dart

```diff
@@ -145,6 +153,9 @@ class NavigationDelegate {
   /// Invoked when a page has finished loading.
   final PageEventCallback? onPageFinished;
 
+  /// Invoked when a page failed loading
+  final PageErrorCallback? onPageError;
```

Comment:
Can you remove this field and just make this a local variable in the constructor. The reason being is that this value can end up not representing the true callback if an app would call the platform implementation setter.